### ISSUE TITLE
[Bug] Open self-posts in browser

### ISSUE THREAD

**Issue by michael-lazar** [michael-lazar/rtv#21]
Referenced by #3, after testing this out I've found a couple of bugs.

* With Firefox 36.0 links open correctly but a warning message is spit out on the screen. This is especially annoying because it writes on top of the text generated by curses.
```
(process:7067): GLib-CRITICAL **: g_slice_set_config: assertion 'sys_page_si e == 0' failed
```

* Fails to open selfposts because `data['url']` is saved as `'(selfpost)'` instead of the actual url.

* We should enable this from inside the Submission view too, if the submission is selected.

**Comment by mekhami** [comment]
Hm, I tried opening self posts with the 'o' command and it just seemed like nothing happened, which I was okay with.

Where is this error message generated? Is it not caught in a Python error?

**Comment by michael-lazar** [comment]
It's generated by the firefox browser itself inside of subprocess.Popen(). I'm working on a patch right now.

**Comment by michael-lazar** [comment]
Fixed the first issue in commit 46be3d6.

**Comment by michael-lazar** [comment]
Remaining issues addresses in f238a07b
